mathics/eval/makeboxes/formatvalues.py

```diff
@@ -183,14 +183,10 @@ def do_format_rational(
     if not isinstance(element, Rational):
         return None
     if form is SymbolFullForm:
```

Comment:
I realize this is not part of the changed code, but I have this nagging feeling that code like `if form is ...`  is not the right way ultimately we should be approaching this.

Reply:
@rocky, feel free to push over all the changes you find useful.

Reply:
Right now, I'd like to defer work on this kind of code until a better handle on MakeBox migration.

Reply:
I guess now is the time for this. Thanks for waiting.